clang/test/SemaSYCL/scheduler_target_fmax_mhz.cpp

```diff
@@ -10,20 +10,16 @@ template <int N>
 [[intel::scheduler_target_fmax_mhz(N)]] void zoo() {}
 
 int main() {
-  // CHECK-LABEL:  FunctionDecl {{.*}}test_kernel1 'void ()'
+  // CHECK:  FunctionDecl {{.*}}test_kernel1 'void ()'
   // CHECK:        SYCLIntelSchedulerTargetFmaxMhzAttr {{.*}}
-  // CHECK-NEXT:   ConstantExpr {{.*}} 'int'
```

Comment:
I think `ConstantExpr ` is no longer generated here because of a difference in how attribute expression is obtained in `addIntelSYCLSingleArgFunctionAttr ` vs `AddOneConstantValueAttr`. The latter calls `VerifyIntegerConstantExpression` which generates this AST.

Reply:
@AaronBallman  @premanandrao Should the `ConstantExpr ` be generated. It isn't consistently generated for IntelSYCL attributes, which is why I removed it here.

Reply:
I think it should be -- see my reasoning in https://github.com/intel/llvm/pull/3134#discussion_r568698663

Reply:
Ok. Thanks for the review! I'll make the change!